packages/core/overlays/core-overlays-browser-internal/src/banners/banners_list.tsx

```diff
@@ -52,7 +50,9 @@ const BannerItem: React.FunctionComponent<{ banner: OverlayBanner }> = ({ banner
   return (
     <div
       data-test-priority={banner.priority}
-      className="kbnGlobalBannerList__item"
+      css={({ euiTheme }) => ({
+        '& + &': { marginTop: euiTheme.size.s },
+      })}
```

Comment:
Not sure if it's actually possible to have "sibling" banner elements.

Reply:
Attempting to test this now.

Reply:
I don't see a way of doing this nor is it documented. I set a banner in my `yml` file, then added one via Advanced Settings... which just overrides the yml. I also can't recall seeing this done, but I recall it being requested. My guess, it was explored but not completed.